CompilerSource/backend/GameData.h

```diff
@@ -137,6 +137,9 @@ struct GameData {
   /// Name of project file. Used in metadata, and later, resource loading.
   std::string filename;
 
+  /// Event names
```

Comment:
This is misleading... this is all known data about all events, not just event names.